src/helpers/xml-builder.js

```diff
@@ -1,2390 +1,2473 @@
+    .up();
+
+const buildTableCellMargins = (margin) => {
+  const tableCellMarFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele(
+    '@w',
+    'tblCellMar'
+  );
+
+  ['top', 'bottom'].forEach((side) => {
+    const marginFragment = buildCellMargin(side, margin / 2);
+    tableCellMarFragment.import(marginFragment);
+  });
+  ['left', 'right'].forEach((side) => {
+    const marginFragment = buildCellMargin(side, margin);
+    tableCellMarFragment.import(marginFragment);
+  });
+
+  return tableCellMarFragment;
+};
+
+const buildTableProperties = (attributes) => {
+  const tablePropertiesFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele(
+    '@w',
+    'tblPr'
+  );
+
+  if (attributes && attributes.constructor === Object) {
+    Object.keys(attributes).forEach((key) => {
+      switch (key) {
+        case 'tableBorder':
+          const { top, bottom, left, right } = attributes[key];
+          if (top || bottom || left || right) {
+            const tableBordersFragment = buildTableBorders(attributes[key]);
+            tablePropertiesFragment.import(tableBordersFragment);
+          }
+          // eslint-disable-next-line no-param-reassign
+          delete attributes.tableBorder;
+          break;
+        case 'tableCellSpacing':
+          if (attributes[key]) {
+            const tableCellSpacingFragment = buildTableCellSpacing(attributes[key]);
+            tablePropertiesFragment.import(tableCellSpacingFragment);
+          }
+          // eslint-disable-next-line no-param-reassign
+          delete attributes.tableCellSpacing;
+          break;
+        case 'width':
+          if (attributes[key]) {
+            const tableWidthFragment = buildTableWidth(attributes[key]);
+            tablePropertiesFragment.import(tableWidthFragment);
+          }
+          // eslint-disable-next-line no-param-reassign
+          delete attributes.width;
+          break;
+      }
+    });
+  }
+  const tableCellMarginFragment = buildTableCellMargins(160);
+  tablePropertiesFragment.import(tableCellMarginFragment);
+
+  // by default, all tables are center aligned.
+  const alignmentFragment = buildHorizontalAlignment('center');
+  tablePropertiesFragment.import(alignmentFragment);
+
+  tablePropertiesFragment.up();
+
+  return tablePropertiesFragment;
+};
+
+const buildTable = async (vNode, attributes, docxDocumentInstance) => {
+  const tableFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele('@w', 'tbl');
+  const modifiedAttributes = { ...attributes };
+  if (isVNode(vNode) && vNode.properties) {
+    const tableAttributes = vNode.properties.attributes || {};
+    const tableStyles = vNode.properties.style || {};
+    const tableBorders = {
+      strokes: { top: 'nil', bottom: 'nil', left: 'nil', right: 'nil' },
+      colors: {},
+    };
+    const tableCellBorders = {};
+    // change the default border settings as its possible borders are not provided to table
+    let borderSize = 0;
```

Comment:
Can we make the defaults a constant referenced from the defaults? I see it in multiple spots. Also add this change to the options and readme. For example `0`, `000000`, `nil`